src/main/java/org/aksw/simba/lemming/metrics/single/nodetriangles/ayz/ListingAyzMetric.java

```diff
@@ -34,32 +35,38 @@ public ListingAyzMetric() {
 
     @Override
     public double apply(ColouredGraph graph) {
-        double delta = 3.0;
-        double threshold = Math.pow(graph.getGraph().getNumberOfEdges(), (delta - 1) / (delta + 1));
-        IntSet highDegreeVertices = new DefaultIntSet(Constants.DEFAULT_SIZE); 
-        		//IntSets.from();
+
+        double r = 2.376;
+        double threshold = Math.pow(graph.getGraph().getNumberOfEdges(), (r - 1) / (r + 1));
```

Comment:
`r` is like a hard coded parameter. It is typically better to have them as `private static final` attribute of the class with a javadoc comment that describes its meaning.

The calculation `(r - 1) / (r + 1)` could be handled in a similar way since it is more or less final as well as soon as we have decided on `r`. So something like
```
private static final double EXPONENT = (r - 1) / (r + 1);
```
with a javadoc comment describing its meaning would be good.

Reply:
Done.